bpdv3/Assets/Scripts/Floor/OnTile.cs

```diff
@@ -28,15 +28,15 @@ public class OnTile : MonoBehaviour
 
         private void Update()
         {
```

Comment:
This looks like the enemy movement will not work as it did with this code commented out.
Can you give some information as to why this was commented out please.

Reply:
The enemy movement did work on my pc withou this

Reply:
thanks for the feedback